app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -123,10 +123,11 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
     * @param lng2
     * @param severity
     * @param filetype
+    * @param inlineCheck
```

Comment:
Let's change it from "inlineCheck" to just "inline". Keep it short and to the point!